controllers/validator.go

```diff
@@ -63,6 +63,14 @@ func (r *DPAReconciler) ValidateDataProtectionCR(log logr.Logger) (bool, error)
 	if _, err := getResticResourceReqs(&dpa); err != nil {
 		return false, err
 	}
+
+	// validate non-admin enable and tech-preview-ack
+	if r.checkNonAdminEnabled(&dpa) {
+		if !(dpa.Spec.UnsupportedOverrides[oadpv1alpha1.TechPreviewAck] == TrueVal) {
+			return false, errors.New("in order to enable the non-admin feature please set dpa.spec.unsupportedOverrides[tech-preview-ack]: true")
```

Comment:
Is valid to change error message to `"in order to enable the non-admin feature please set dpa.spec.unsupportedOverrides[tech-preview-ack]: 'true'"` to avoid users telling us they can not input bool here?

Reply:
Done, updated the PR in f7e2b1200ff8d53359b7585a1f087b87f9d0bccd